spec/lib/open_food_network/user_balance_calculator_spec.rb

```diff
@@ -28,14 +28,21 @@ module OpenFoodNetwork
                          state: "completed")
       }
 
+      before do
+        # Sanity check the order
+        expect(o1.total).to eq 13
+        expect(o1.ready_to_ship?).to eq true
+        expect(o1.paid?).to eq true
+      end
```

Comment:
I really dislike this sort of test code. I see doubts when I read that and that undermines my confidence. It feels like we don't trust what the test sets up and so we have the need to check for a known state. I'd rather spend the time to pull the test setup apart, understand what's the state we are setting things in, and write it in a way that increases our confidence in it.

EDIT alright, this class will hopefully be gone in a month's time or so. My point is still valid but it's not worth spending any more time here.